src/App.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { Component, ClassAttributes } from "react";
```

Comment:
Optional: import `ClassAttributes` as a type to remove ambiguity. This won't be needed when refactored as a functional component.

```suggestion
import { Component, type ClassAttributes } from "react";
```